src/tracker.rs

```diff
@@ -199,7 +165,7 @@ impl ProgressSet {
         let pr = Progress::new(next_idx, max_inflight);
         for id in &meta.conf_state.as_ref().unwrap().voters {
             self.progress.insert(*id, pr.clone());
-            self.configuration.voters.insert(*id);
+            self.configuration.voters.voters.insert(*id);
```

Comment:
This looks wired... Is there potential extra fields in `Configuration` besides `voters` ?

Reply:
Yes. It will be added when Joint configuration is ported.